plugins/woocommerce-blocks/assets/js/blocks/product-gallery/inner-blocks/product-gallery-thumbnails/edit.tsx

```diff
@@ -51,7 +51,7 @@ export const Edit = ( { attributes, setAttributes, context }: EditProps ) => {
 					);
 				} ) }
 			</div>
-		) : null;
+		);
 	};
 
 	return (
```

Comment:
We're passing `attributes` and `setAttributes` down as props to `<ProductGalleryThumbnailsBlockSettings />` but [I don't think this component expects these props](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-blocks/assets/js/blocks/product-gallery/inner-blocks/product-gallery-thumbnails/block-settings/index.tsx#L50)

Reply:
Good catch!